tests/unittests/Foundation/ReferenceFoundation/TestNSArray.mm

```diff
@@ -150,13 +150,19 @@ static int objectIndexInArray(NSArray* array, int value, int startingFrom, int l
     //        NSBinarySearchingOptions* searchForBoth = NSBinarySearchingFirstEqual | NSBinarySearchingLastEqual
     //        auto _ = objectIndexInArray(array, 9, 0, 13, searchForBoth)
 
-    auto notFound = objectIndexInArray(array, 11, 0, 13);
-    ASSERT_EQ_MSG(notFound, NSNotFound, @"NSArray return NSNotFound if object is not found.");
```

Comment:
What @msft-Jeyaram  and @ms-jihua have said is correct. That said, I definitely agree that it is *not* very clear at the moment. @ms-jihua and I talked, and she will add a header to each of the ReferenceFoundation files to make this hopefully more clear in the future. Sorry for the confusion and thank you for putting up with us :-)

---
In reply to: [67920136](https://github.com/Microsoft/WinObjC/pull/569#discussion_r67920136) [](ancestors = 67920136,67917856)